### PR TITLE
Split "Managing Security Compliance" into a new guide

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; planned orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.4 or older.

### CODE REVIEW

File: guides/doc-Managing_Security_Compliance/master.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+= {ManagingSecurityDocTitle}
+
+ifdef::foreman-deb[]
+This feature is not available for Debian/Ubuntu.
+// Because we don't package foreman_openscap for Debian
+endif::[]
+ifndef::foreman-deb[]
+
+Security compliance management is the ongoing process of defining security policies, auditing systems for compliance with those policies and resolving instances of non-compliance.
```
Comment (maximiliankolb) [inline guides/doc-Managing_Security_Compliance/master.adoc:14]: Kindly asking you to move this introduction to either a snippet, or, even better, along with the heading to a "con_" file. Reason: in downstream, I don't use the "master.adoc" or "assembly_" files for various reasons.

Comment (Lennonka) [reply guides/doc-Managing_Security_Compliance/master.adoc]: Fixed. Compromise, because I want to stick with the same style as in other `master.adoc`s.

**Review by maximiliankolb — APPROVED** [review]
The diff looks good to me & I have compared https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#Managing_Security_Compliance_admin with https://theforeman-foreman-documentation-preview-pr-2208.surge.sh/nightly/Managing_Security_Compliance/index-katello.html which also LGTM!

**Comment by Lennonka** [comment]
Cherry-picked:
- 707989a5a..cc73c65a4  3.7 -> 3.7
- eb8bd3352..b7f1fa774  3.6 -> 3.6

**Pull request merged**